packages/core-mobile/app/new/components/navigation/BlurredBackgroundView.tsx

```diff
@@ -1,16 +1,8 @@
 import React from 'react'
-import { useColorScheme } from 'react-native'
 import { GlassView } from '@avalabs/k2-alpine'
 
 const BlurredBackgroundView = (): JSX.Element => {
-  const colorScheme = useColorScheme()
-
-  return (
-    <GlassView
-      style={{ flex: 1 }}
-      glassType={colorScheme === 'dark' ? 'dark' : 'light'}
-    />
-  )
+  return <GlassView style={{ flex: 1 }} />
```

Comment:
If you set the glassType, it applies a background color to the blur, which causes a color difference with the backgroundView. Therefore, we don’t use glassType for the blurView used as the background for the header and tab bar.